packages/host/src/components/Initialise/hooks.ts

```diff
@@ -0,0 +1,130 @@
+import { useEffect, useState } from 'react';
+import create from 'zustand';
+import { AppRoute } from '@openmsupply-client/config';
+import {
+  AuthenticationError,
+  AuthError,
+  LocalStorage,
+  ServerStatus,
+  useNavigate,
+} from '@openmsupply-client/common';
+import { useHost } from '../../api/hooks';
+
+const SERVER_RESTART_TIMEOUT = 90000;
+const POLLING_INTERVAL = 3000;
```

Comment:
polling every 3s : originally was 1s but was concerned that the queries were delaying the server restart process! Given how long the whole process takes, thought that a few seconds extra doesn't matter

Reply:
I just played with it and increasing the value makes it faster, the query seems to slow down the server from shutting down gracefully...

Think best would be to wait some time before start polling. Think without ongoing connections the server restart is actually quite fast. I tried:

```js
    setTimeout(() => {
      setIsPolling(true);
    }, 6000);
```
and this was ok

Reply:
😁 ah, thanks. I wondered about doing that and didn't actually test it. Have added it in